.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Summary**
+
+Describe the bug here. What did you expect to happen (e.g. a request) and what did you get that was unexpected?
+
+**OpenFGA version or commit**
+
+If you know it, enter here.
+
+**Store data**
+
+If relevant, provide information about your authorization model and tuples. For example:
+
+```
+model-file: |
+  model
+    schema 1.1
+  type user
+  type organization
+    relations
+      define member: [user]
+tuples:
+  - user: user:anne
+    relation: member
+    object: organization:openfga
+  - user: user:bob
+    relation: member
+    object: organization:openfga
```

Comment:
Can we also provide as an example check/list-tuples tests, so they can demonstrate how the behavior is not what they expect?

Reply:
My hope was that they put that in the summary section. But i'll add it here.